src/population_loader.rs

```diff
@@ -0,0 +1,99 @@
+use crate::parameters::Parameters;
+use ixa::{
+    context::Context, define_person_property, define_person_property_with_default, error::IxaError,
+    global_properties::ContextGlobalPropertiesExt, people::ContextPeopleExt,
+};
+
+use serde::Deserialize;
+use std::path::PathBuf;
+
+#[derive(Deserialize, Debug)]
+#[allow(non_snake_case)]
+pub struct PeopleRecord<'a> {
+    age: u8,
+    homeId: &'a [u8],
+}
+
+define_person_property!(Age, u8);
```

Comment:
I wonder if by the same logic that we have indexing the property going in `main`, should we define the property there too? I feel like it is most logical to have the property defined here, but just thinking about best structure. Would be curious what others think.

Reply:
Given that we've designed ixa (including person properties/person property initialization) to support a modular structure, I think it would be better to define properties with the code they're related to v.s. trying to stick everything in main.